<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsaGameDialog</class>
 <widget class="QDialog" name="CsaGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通信対局（CSA）</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <widget class="QGroupBox" name="groupBoxPlayer">
     <property name="title">
      <string>こちら側の対局者</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutPlayer">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutHuman">
        <item>
         <widget class="QRadioButton" name="radioButtonHuman">
          <property name="text">
           <string>人間</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerHuman">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutEngine">
        <item>
         <widget class="QRadioButton" name="radioButtonEngine">
          <property name="text">
           <string>エンジン</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEngine">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEngineSettings">
          <property name="text">
           <string>エンジン設定...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerEngine">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxServer">
     <property name="title">
      <string>サーバー</string>
     </property>
     <layout class="QFormLayout" name="formLayoutServer">
      <item row="0" column="0">
       <widget class="QLabel" name="labelHistory">
        <property name="text">
         <string>履歴から選ぶ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxHistory">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelVersion">
        <property name="text">
         <string>バージョン</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxVersion">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>CSAプロトコル1.2.1 読み筋コメント出力あり</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CSAプロトコル1.2.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CSAプロトコル1.1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelHost">
        <property name="text">
         <string>接続先ホスト</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditHost">
        <property name="placeholderText">
         <string>例: 172.17.0.2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>ポート番号</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxPort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>4081</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelId">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditId">
        <property name="placeholderText">
         <string>ログインID</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>パスワード</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>パスワード</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBoxShowPassword">
        <property name="text">
         <string>パスワードを表示する</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QToolButton" name="toolButtonFontDecrease">
       <property name="text">
        <string>A-</string>
       </property>
       <property name="toolTip">
        <string>フォントサイズを小さくする</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFontIncrease">
       <property name="text">
        <string>A+</string>
       </property>
       <property name="toolTip">
        <string>フォントサイズを大きくする</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerButtons">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>対局開始</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>キャンセル</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
